### DISCUSSION TITLE
Właściwości, style CSS i kategorie bloków Gutenberg

### DISCUSSION THREAD

**Discussion by giscus** [robert-orlinski/robertorlinski.pl#20] [Komentarze]
# Właściwości, style CSS i kategorie bloków Gutenberg

W drugiej części serii WordPress Gutenberg, zajmiemy się właściwościami, kategoriami oraz stylami CSS, własnych bloków Gutenberga 💥

https://robertorlinski.pl/artykuly/wordpress-gutenberg-002

**Comment by bibliotekarz** [comment]
skoro gdzieś tam piszesz że nie prowadzisz statystyk do daje znać że ktoś czyta z zaciekawieniem Twoje posty :D
Pewnie poza wordpressa nie wyjdę więc jestem małoprzyszłościowym odbiorcą ale wiedz że wykonana dobra robota się nie marnuje.

**Reply by robert-orlinski** [reply]
I tak super słyszeć! 😄 Cieszę się, że te posty dalej dają wartość :)
